app/presenters/admin_access_presenter.rb

```diff
@@ -18,31 +18,47 @@ def permitted_access_levels
     UserAccess::LEVELS.slice(*admin.permitted_access_levels).values
   end
 
-  memoize def access_tree
-    UserAccessTree.new(admin)
-  end
-
-  delegate :visible?, to: :access_tree
-
   def visible_access_tree
     if admin.access_across_organizations?(:any)
       {
-        data: access_tree.organizations,
+        data: organization_tree,
         render_partial: "email_authentications/invitations/organization_access_tree",
         root: :organization
       }
     elsif admin.access_across_facility_groups?(:any)
       {
-        data: access_tree.facility_groups,
+        data: facility_group_tree,
         render_partial: "email_authentications/invitations/facility_group_access_tree",
         root: :facility_group
       }
     else
       {
-        data: access_tree.facilities,
+        data: visible_facilities,
         render_partial: "email_authentications/invitations/facility_access_tree",
         root: :facility
       }
     end
   end
+
+  memoize def organization_tree
```

Comment:
Removed the `UserAccessTree` class in favor of these tiny 3-4 methods that now power the access trees.